templates/global/filter_form.html

```diff
@@ -1,52 +1,48 @@
 {% block filter_form %} {% load static %}
 <h3 class="text-center text-black pt-1">House Filter</h3>
 
-    <form
-            method="post"
-            id="HouseFilter"
-            novalidate
-    >
-        <table>
-            {{ form.media.js }}
-            {% for field in form %}
-                <tr>
-                    {%if not "income" in field.label_tag%}
-                        <th>{{ field.label_tag }}</th>
-                        <td colspan="2">{{ field }}</td>
-                    {% endif %}
-                </tr>
-            {% endfor %}
-            <tr style = "border:none">
-                <th colspan="3"><label>Income:</label></th>
-            </tr>
+<form method="post" id="HouseFilter" novalidate>
```

Comment:
why did you change this file?

Reply:
I didn't maybe when lior helped me or it has done by mistake.
Lior told me that something that is not in my files needed to be changed.
Did you write this code? do you need it?

Reply:
This is code that was in Lior's PR

Reply:
no code was supposed to change here 
you reformatted code that you didn't change by mistake please change it back

Reply:
I ran pre commit and it is reformating it automatically

Reply:
I don't think the format is really critical am I wrong?